### ISSUE TITLE
Axis inversion

### ISSUE THREAD

**Issue by dennemark** [proj4js/proj4js#492]
Hi,
I am running into an issue where proj4 does not invert the axis. 
The EPSG is EPSG:2398.
I am using the 1. description. Actually it differs from the 2. taken from epsg.io slightly, but i guess this won't make the difference
1. `+proj=tmerc +lat_0=0 +lon_0=12 +k=1 +x_0=4500000 +y_0=0 +ellps=krass +towgs84=26,-121,-78,0,0,0,0 +units=m +no_defs`
2. `+proj=tmerc +lat_0=0 +lon_0=12 +k=1 +x_0=4500000 +y_0=0 +ellps=krass +towgs84=24,-123,-94,-0.02,0.25,0.13,1.1 +units=m +no_defs +type=crs`

I am converting a [lng, lat] coordinate  from 'EPSG:4326' to the target system. The resulting coordinate should be [projectedLat, projectLng] though.
proj4('EPSG:4326', target, [lng, lat]) 

I read about this issue for the given coordiante system on a german blog post, where WFS-server would not invert the axis for the old 'EPSG:2398' string, but would do it for the newer urn version 'urn:ogc:def:crs:EPSG::2398' to have backwards compatibility. In our case we use the urn version, since the old approach is not always supported by servers, so the urn version is more reliable.
https://www.weichand.de/2013/03/24/wfs-achsenreihenfolge-von-koordinatensystemen/

**tldr**: proj4 should convert [lng, lat] to [projLat, projLng] for a coordinate system that needs axis inversion

I read about some old issues with axis inversion, but i am not sure if it is related https://github.com/proj4js/proj4js/issues/352

Kind regards!

**Comment by dennemark** [comment]
Thanks a lot! I missed the right key word. Going to try it!

**Comment by ahocevar** [comment]
I recommend https://spatialreference.org/ as source for projection definitions, and use the WKT v1 format. With that, it should work as you expect: https://spatialreference.org/ref/epsg/2389/ogcwkt
